kleros-app/src/lib/atlas/providers/AtlasProvider.tsx

```
@@ -0,0 +1,343 @@
+      try {
+        if (!address || !chainId) return;
+        setIsSigningIn(true);
+        const nonce = await getNonce(atlasGqlClient, address);
+
+        const message = createMessage(address, nonce, chainId, statement);
+        const signature = await signMessageAsync({ message });
+
+        const token = await loginUser(atlasGqlClient, { message, signature });
+        setAuthToken(token);
+      } catch (err: unknown) {
+        throw err;
```

Biome (1.9.4)

[error] The catch clause that only rethrows the original error is useless.

An unnecessary catch clause can be confusing.
Unsafe fix: Remove the catch clause.


(lint/complexity/noUselessCatch)